packages/dev/core/src/Lights/shadowLight.ts

```diff
@@ -387,4 +387,11 @@ export abstract class ShadowLight extends Light implements IShadowLight {
         }
         return this;
     }
+
+    /** @hidden */
+    protected _syncParentEnabledState() {
+        super._syncParentEnabledState();
+        (this.transformedPosition as any) = null;
+        (this.transformedDirection as any) = null;

```

Comment:
I think you should set these properties only if `this.parent` is `null`, or maybe, alternatively, keep those lines and call `computeTransformedInformation()` to update the `transformedPosition` / `transformedDirection` properties (if necessary)?